web/lib/helpers/helpers.ts

```diff
@@ -93,3 +130,34 @@ export function pick<
     keys.map(key => [key, obj[key as unknown as keyof T]])
   ) as { [key in K]: key extends keyof T ? T[key] : undefined }
 }
+
+export function formatPhoneNumber(value: string) {
+  // Remove any existing spaces and non-numeric characters
+  const phoneNumber = value.replace(/\D/g, '')
+  // Start with +
+  const startsWithPlus = value.startsWith('+')
+  // Limitation
+  const maxDigits = startsWithPlus ? 12 : 9
+  const limitedPhoneNumber = phoneNumber.slice(0, maxDigits)
+  // Add spaces after every third digit
+  const formattedPhoneNumber = limitedPhoneNumber.replace(/(\d{3})(?=\d)/g, '$1 ')
+  return startsWithPlus ? `+${formattedPhoneNumber}` : formattedPhoneNumber || ""
+}
+
+// Get rid of anything that isn't letter or spaces, and replace redundant spaces by one space and trim spaces from front.
+export function formatName(value: string) {
+  return XRegExp.replace(value, XRegExp('[^\\p{L}\\s]+', 'g'), '').replace(/\s+/g, ' ').trimStart()
```

Comment:
Na tohle nepotřebujeme regex balíček stačí normální regex z js

Reply:
Máš pravdu. Když jsem to psala, tak to bylo o něco složitější a nějak se mi to nedařilo zapsat pomocí js regex (myslím, že jsem zapomínala na /u flag pro unicode), postupně se to ale zjednodušilo a tohle není už problém napsat pomocí obyčejného regex. Každopádně jsem se tu část s odstraněním všech charakterů co není písmeno ('[^\\p{L}\\s]+', 'g'), '') nebo v js regex .replace(/[^\p{L}\s]+/gu, '') rozhodla odstranit úplně. Nechala jsem jen odstraňování přebytečných mezer. Ono asi není potřeba být tak šikanující, kor například francouzská jména mají apostrofy a nebo se někdo jmenuje X Æ A-12, tak to není třeba takhle řešit.